tests/sentry/lang/javascript/test_plugin.py

```diff
@@ -315,8 +315,11 @@ def test_expansion_via_release_artifacts(self):
             headers={'Content-Type': 'application/json'},
         )
         f2.putfile(open(get_fixture_path('file1.js'), 'rb'))
+
+        # Intentionally omit hostname - use alternate artifact path lookup instead
+        # /file1.js vs http://example.com/file1.js
         ReleaseFile.objects.create(
-            name='http://example.com/{}'.format(f2.name),
+            name='/{}'.format(f2.name),
```

Comment:
I'd add a test case that covers a match over both a full url and a relative url.